saleor/order/actions.py

```diff
@@ -40,13 +50,29 @@
 )
 
 if TYPE_CHECKING:
-    from ..account.models import User
     from ..warehouse.models import Warehouse
-    from .models import Order
 
 logger = logging.getLogger(__name__)
 
 
+OrderLineIDType = int
+QuantityType = int
+
+
+@dataclass
+class OrderLineData:
+    line: OrderLine
+    quantity: int
+    replace: bool = False
+
+
+@dataclass
+class FulfillmentLineData:
+    line: FulfillmentLine
+    quantity: int
+    replace: bool = False
```

Comment:
On the "checkout performance" branch we keep similar dataclasses in `order/__init__.py`, maybe we should move these as well? What do you think @fowczarek ?

Reply:
It's will be great to have all data classes in the same place for each module.

Reply:
So in that case, it shouldn't be a file like `dataclasses.py` or something? I worry that `__init__` can be like `utils.py`